src/components/Admin/QuestionDisplay.js

```diff
@@ -0,0 +1,212 @@
+import React, { Component, Fragment } from "react";
+import styled from "styled-components";
+
+import Button from "react-bootstrap/Button";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+
+import { withFirebase } from "upe-react-components";
```

Comment:
This component shouldn't have access to firebase, as it should be a functional/presentational component that just shows data passed to it through props. References to firebase like in mounting etc, should be removed. To edit the data, it should receive a function to change it from it's parent (which you've already done here).